src/PixiEditor/Exceptions/RecoverableException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Runtime.Serialization;
+
+namespace PixiEditor.Exceptions;
+
+public class RecoverableException : Exception
```

Comment:
Maybe it would be good to add localization key to this, so exception can be in english, but notice dialog would use Exception.LocalizedKey or sth